apps/meteor/packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1753,6 +1754,8 @@
   "Enable_CSP": "Enable Content-Security-Policy",
   "Enable_CSP_Description": "Do not disable this option unless you have a custom build and are having problems due to inline-scripts",
   "Enable_Desktop_Notifications": "Enable Desktop Notifications",
+  "Enable_Feedback_Survey": "Enable Feedback Survey",
+  "Enable_Feedback_Survey_Description": "Enable the feedback survey to be shown to users",
```

Comment:
Can we change this description please:
"Enable the feedback survey monitor to support the Dialogflow custom payload that indicates a user should be presented with the survey after inactivity."